examples/training/semantic_segmentation/pascal_voc/basic_training.py

```diff
@@ -136,15 +148,16 @@ def proc_eval_fn(examples):
         boundaries=[30000 * 16 / global_batch],
         values=[base_lr, 0.1 * base_lr],
     )
-    backbone = ResNet50V2(
+    backbone = models.ResNet50V2(
         include_rescaling=True,
         # This argument gives a 2% mIoU increase
         stackwise_dilations=[1, 1, 1, 2],
         input_shape=(512, 512, 3),
         include_top=False,
         weights="imagenet",
     )
-    model = DeepLabV3(classes=21, backbone=backbone)
+    model = segmentation.__dict__[FLAGS.model_name]
```

Comment:
this can be retrieved from keras_cv.models.__dict__, no?

Reply:
Good catch! They weren't originally in `models`, but have been moved since

Reply:
Updated